src/components/Highlight/index.tsx

```diff
@@ -1,15 +1,30 @@
 import styled from "@emotion/styled";
 import { palette } from "@leafygreen-ui/palette";
 
-const { red } = palette;
+const { red, green, blue, yellow, purple } = palette;
 
-const Highlight = styled.mark<{ "data-cy"?: string }>`
-  background-color: ${red.light2};
+const Highlight = styled.mark<{ "data-cy"?: string; color?: string }>`
+  background-color: ${({ color }) => color};
   font-weight: bold;
 `;
 
 Highlight.defaultProps = {
   "data-cy": "highlight",
+  color: red.light2,
 };
 
+const highlightColorList = [
+  green.light1,
+  blue.light2,
+  red.light3,
```

Comment:
This color might be a candidate for removal, as it can't be seen when a line is the active search line